OpenRA.Game/Game.cs

```diff
@@ -172,6 +172,38 @@ internal static void StartGame(string mapUID, WorldType type)
 			GC.Collect();
 		}
 
+		public static void RestartGame()
```

Comment:
I think it's weird to have this Game.RestartGame() in the same file as Game.StartGame() which works completely differently to it, and StartMission() (the real corresponding method to RestartGame()) way out in MissionBrowserLogic.

I'd like to see RestartGame() and StartMission() moved together if possible.